components/director/internal/domain/labeldef/repository.go

```diff
@@ -34,10 +34,7 @@ func (r *repo) Create(ctx context.Context, def model.LabelDefinition) error {
 		return errors.Wrap(err, "while converting Label Definition to insert")
 	}
 
-	columns := []string{"id", "tenant_id", "key"}
-	if entity.SchemaJSON.Valid {
-		columns = append(columns, "schema")
-	}
+	columns := []string{"id", "tenant_id", "key", "schema"}
```

Comment:
What about moving it to the const `labelDefinitionFields` like is done in runtime `repository`